### ISSUE TITLE
Treat W3002 as informational or provide a suggested workaround

### ISSUE THREAD

**Issue by mjs510** [aws-cloudformation/cfn-lint#2297]
cfn-lint version: 0.61.1

W3002 is described as "Some properties can be configured to only work with the CloudFormationpackage command. Warn when this is the case so user is aware."

It's not clear what someone is meant to do if they are aware and this is the intended outcome.

When running cfn-lint as a pre-commit hook or as a github action etc, this warning will fail the build unless it is explicitly disabled. Would it be better to change this to an informational code?

**Comment by PatMyron** [comment]
[added mid-2018](https://pypi.org/project/cfn-lint/#history) in [`0.7.0`](https://github.com/aws-cloudformation/cfn-lint/blob/main/CHANGELOG.md#070) before informational rules were added in [`0.8.0`](https://github.com/aws-cloudformation/cfn-lint/blob/main/CHANGELOG.md#080): https://github.com/aws-cloudformation/cfn-lint/pull/315

hesitant to change rule IDs since people are already ignoring existing rule IDs
(maybe rule severity should have been separate from rule IDs in retrospect)

**Comment by mjs510** [comment]
Does it matter if a warning that is already ignored changes to an informational message? Presumably the overall effect would be the same?

**Comment by CraigHead** [comment]
We upgraded to the latest cfn-lint (`0.66.0`) today and now we are noticing that `AWS::CloudFormation::Stack` resources are complaining 
with 
```
W3002 This code may only work with `package` cli command as the property (Resources/DatadogIntegrationRoleStack/Properties/TemplateURL) is a string
```

However I did some testing and found out that this check is being flagged on strings that are not Pseudo-functions such as `!Sub`.

For example:
This causes the check to be tripped
```yaml
  NestedStack:
    Type: AWS::CloudFormation::Stack
    Properties:
      TemplateURL: "https://bucket-name.s3.amazonaws.com/path/nestedStackTemplate.yaml"
```

but this does not
```yaml
  NestedStack:
    Type: AWS::CloudFormation::Stack
    Properties:
      TemplateURL: !Sub "https://bucket-name-${AWS::Region}.s3.amazonaws.com/path/nestedStackTemplate.yaml"
```

This appears to be a regression over `cfn-lint 0.65.0`

**Comment by stevehaneytrailblazer** [comment]
Could there be a CLI switch to ignore warnings so that CI/CD is unaffected unless there are errors?

**Comment by kddejong** [comment]
Looking into the different issues that get to the same request.  Also look at https://github.com/aws-cloudformation/cfn-lint/issues/1586#issuecomment-1286419486

**Comment by kddejong** [comment]
Given the ability to suppress this warning I'm going to close this issue as this falls into the warning category.  Warnings are meant to point to possible limitations in the template which is met by the fact that there is limited deployment methodologies for this template.   For the non zero exit code we will track under #1586
